core/physical_substances.mcf

```diff
@@ -0,0 +1,114 @@
+# Schema for all chemical elements and compounds
+
+Node: dcid:PhysicalSubstance
+typeOf: schema:Class
+name: Physical Substance
+description: All types of substances found on earth like elements, compounds belonging to all three states of matter (solid, liquid and gas)
+
+Node: dcid:Element
+typeOf: schema:Class
+name: Element
+subClassOf: dcs:PhysicalSubstance
+description: Elements in the periodic table
+
+Node: dcid:Compound
+typeOf: schema:Class
+name: Compound
+subClassOf: dcs:PhysicalSubstance
+description: Compounds are physical substances that contain more than one element. Compounds can be homogeneous (E.g.: O2) or heterogeneous (E.g.: H2O).
+
```

Comment:
Can we re-use [`ChemicalCompound`](https://autopush.datacommons.org/browser/ChemicalCompound) instead of defining a new node called `dcid:Compound`?